nautobot_netbox_importer/tests/fixtures/3.7/summary.json

```diff
@@ -11102,28 +11766,15 @@
             "nautobot_flags": "DiffSyncModelFlags.SKIP_UNMATCHED_DST",
             "default_reference_uid": null,
             "imported_count": 0,
-            "fields": {
-                "id": {
-                    "name": "id",
-                    "nautobot_name": "id",
-                    "nautobot_internal_type": "AutoField",
-                    "nautobot_can_import": true,
-                    "importer": null,
-                    "definition": "id",
-                    "from_data": false,
-                    "is_custom": true,
-                    "default_value": null,
-                    "disable_reason": ""
-                }
-            }
+            "fields": {}
         },
-        "users.adminuser": {
-            "content_type": "users.adminuser",
-            "content_type_id": 103,
+        "users.netboxuser": {
```

Comment:
Does this map to Nautobot's `users.user`?

Reply:
Seems like it's just a proxy model without any data in the DB.